### ISSUE TITLE
Remove requirement for user authentication?

### ISSUE THREAD

**Issue by AndLindemann** [devfake/flox#141]
It seems most modifications/additions in the app can only be made when the admin user is logged in. Is there an option to remove that requirement? 
I don't really need users to be logged in to do anything, don't think most users would expose this to the internet and for home-internal use it probably doesn't really need user auth/password?
It's not like someone accidentally stumbling onto the app could cause any serious trouble - and if yes, you shouldn't really let him into your home network anyway. ;)

**Comment by devfake** [comment]
That's interesting. The implementation for that would be quite easily. If you could wait until the weekend, I can do it :)

**Comment by AndLindemann** [comment]
Sure, take the time. It was just a suggestion as I stumbled upon it while wondering why I cannot add items to the watchlist while on mobile.
If you're not logged in, searching in the search bar will not show any TMDB results. So that was quite confusing as it was working fine on my desktop (where I was logged in). I didn't really expect that this feature would be limited by a logged in user.
I believe ideally I would want a user authentication only to access the settings pages.
Everything else, like rating movies and adding new ones, I would at least like to see the option to be available without authentication.

Just wanted to add: It's a great app otherwise. Nice visuals. I was looking for something like this as my wife wants to have an easy to use thing where she can add movies we'd want to watch and also do some simple rating of things we had seen. So while searching I found Flox and it does exactly this in a really easy way. So good job!
